uat/test_uat_build.py

```diff
@@ -35,10 +39,40 @@ def test_build_template_zip(change_test_dir, gdk_cli):
         .joinpath("HelloWorld.zip")
         .resolve()
     )
-
     assert artifact_path.exists()
 
 
+def test_build_template_zip_fail_with_no_artifact_since_v1_0_0(change_test_dir, gdk_cli):
```

Comment:
instead of adding `since_` here, can we mark it as `min='1.0.0'`?

Reply:
done